CONTRIBUTING.md

```diff
@@ -649,11 +649,11 @@ should be canceled.
 
 ### Maintainers
 
-- [Damien Mathieu](https://github.com/dmathieu), Elastic
+- [Damien Mathieu](https://github.com/dmathieu), Elastic ([GPG](https://keys.openpgp.org/search?q=408B8E44F0873832)
```

Comment:
Should we use the full fingerprint?

Per Arch's [PKGBUILD](https://wiki.archlinux.org/title/PKGBUILD)

> Only full fingerprints are accepted. They must be uppercase and must not contain whitespace characters.